federation/manifests/federated-apiserver-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{.FEDERATED_APISERVER_DEPLOYMENT_NAME}}
+  namespace: federation-e2e
+  labels:
+    app: federation-e2e
```

Comment:
I was hoping that we can keep these manifests generic, so that I can use them to bring up a local federation control place or can use them in e2e. So will be great if we can get rid of all the e2e references in these yaml files.